html/appetizers.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Berkshire+Swash&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Asian Kitchen</title>
+
+    <link rel="stylesheet" href="../styles.css" />
+  </head>
+  <div id="home">
+    <a class="alignleft" href="../index.html">Home</a>
+  </div>
```

Comment:
Since you are using CSS, I think you could make the A tag a block element instead of using the Div. However looking further down into other files, I see that you group multiple a tags together, guess it is fine then